packages/flutter/lib/src/widgets/system_context_menu.dart

```diff
@@ -458,5 +458,65 @@ final class IOSSystemContextMenuItemLiveText extends IOSSystemContextMenuItem {
   }
 }
 
-// TODO(justinmc): Support the "custom" type.
-// https://github.com/flutter/flutter/issues/103163
+/// Creates an instance of [IOSSystemContextMenuItem] for custom action buttons
+/// defined by the developer.
+///
+/// Only supported on iOS 16.0 and above.
+///
+/// The [title] and [onPressed] callback must be provided.
+///
+/// {@tool dartpad}
+/// This example shows how to add custom menu items to the iOS system context menu.
+///
+/// ** See code in examples/api/lib/widgets/system_context_menu/system_context_menu.1.dart **
+/// {@end-tool}
```

Comment:
Do you think it would be valuable to also add this example to SystemContextMenu.items, or is it too much?